src/external/dataCenterGateway/gql/QUERY_SAMPLE_MATCHED_ANALYSIS_BY_DONOR.TS

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020 The Ontario Institute for Cancer Research. All rights reserved
+ * Copyright (c) 2022 The Ontario Institute for Cancer Research. All rights reserved
```

Comment:
changing the year only in one file? seems unlikely of you.
thought you'd advocate for one PR updating the year in all files...

Reply:
😆 nor did I think you would or should!

you know imo, small regular improvements are better than delayed and seldom large ones.